src/App/App.js

```diff
@@ -66,6 +66,17 @@ class App extends Component {
         })
   }
  
+  hoverMovie = (id) => {
```

Comment:
Function to search within this.state.movies and update this.state.movie to contain the relevant info